docs/components/modeler/desktop-modeler/element-templates/using-camunda-connector-templates.md

```diff
@@ -0,0 +1,28 @@
+---
+id: using-camunda-connector-templates
+title: Using Camunda Connector templates
+description: "Learn how to use Camunda Connector templates in Desktop Modeler."
+---
+
+Users of Camunda 8 and Desktop Modeler can use the same [Connector templates](../../../connectors/out-of-the-box-connectors/available-connectors-overview.md) that are available in Web Modeler.
```

Comment:
```suggestion
You can enable automatic fetching of [Camunda Connector templates](../../../connectors/out-of-the-box-connectors/available-connectors-overview.md) in the Desktop Modeler. This will make all out-of-the-box templates available for direct use.

Alternatively connector templates may be configured globally or per project following the standard configuration mechanisms for element templates.  
```